t/135-worker-id.t

```diff
@@ -31,3 +31,38 @@ GET /lua
 --- no_error_log
 [error]
 --- skip_nginx: 3: <=1.9.0
+
+
+
+=== TEST 2: worker id nil on no worker process
+--- http_config 
+    proxy_buffering on;
+    proxy_cache_valid any 10m;
+    proxy_cache_path /data1/cache levels=1:2 keys_zone=my-cache:8m max_size=1000m inactive=600m;
```

Comment:
It's wrong to use system-level path like `/data1/cache` since it requires human intervention and pollutes the current system running this test.